content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -3259,7 +3258,7 @@ Here's what you need to know about **pushes**:
 
 ## Chats
 
-### `incoming_chat_thread`
+### `incoming_chat`
 Informs about a new thread coming in the chat. The push payload contains not only the new thread, but the whole chat data structure. If the chat was started with some initial events, the thread object contains them.  
```

Comment:
Maybe, we should add more explanation here as well, like in Close Thread -> Deactivate Chat. 
"From the UI perspective, a new chat starts thus the method name."

Reply:
Ok, I have changed it a little bit.